elibrary-core/source/src/controller/java/com/javafee/elibrary/core/tabbedform/TabLoanServiceEvent.java

```diff
@@ -318,14 +319,18 @@ private void onClickBtnPenalty() {
 				HibernateUtil.getSession().update(lend);
 				HibernateUtil.commitTransaction();
 
-				((LoanTableModel) tabbedForm.getPanelLoanService().getLoanTable().getModel()).reloadData();
-				((VolumeTableModel) tabbedForm.getPanelLoanService().getVolumeLoanTable().getModel()).reloadData();
-				JOptionPane.showMessageDialog(tabbedForm.getFrame(), "Kara została spłacona", "Spłacona",
+				JOptionPane.showMessageDialog(tabbedForm.getFrame(),
```

Comment:
((LoanTableModel) tabbedForm.getPanelLoanService().getLoanTable().getModel()).reloadData();
((VolumeTableModel) tabbedForm.getPanelLoanService().getVolumeLoanTable().getModel()).reloadData();

Should be reverted.